cmw-dto/src/main/java/com/liberologico/cloudesire/cmw/model/dto/ProductDTO.java

```diff
@@ -197,6 +197,12 @@
     @ApiModelProperty( "Authentication header value for the upstream API" )
     private String apiHeaderValue;
 
+    public ProductDTO( String name, String identifier, ProductType type, UrlEntityDTO company )
```

Comment:
why not make a new DTO extending ProductDTO with only this constructor?

Reply:
because regular products too can be created by admin, so it's useful here as well